src/Task/Phpunit.php

```diff
@@ -21,12 +21,22 @@ public static function getConfigurableOptions(): OptionsResolver
             'testsuite' => null,
             'group' => [],
             'always_execute' => false,
+            'order' => 'default',
```

Comment:
This will always set the order to default. What happens if you specified an order in your phpunit task? Will it fall back to that one or use the default of phpunit?
Maybe better to set it to `null` by default so that you can overwrite when it makes sense to do so?